src/models/style-model/index.js

```diff
@@ -0,0 +1,122 @@
+import DEFAULTS from "../../style-defaults";
+
+export default function createStyleModel({ layout, themeService, flags }) {
+  const themeStyle = themeService.getStyles();
+
+  const findComponent = (key) => layout.components?.find((o) => o.key === key) ?? undefined;
+
+  const resolveStyle = (component, componentValue, styleValue, defaultValue) => {
+    if (!flags?.isEnabled("IM_5036_ORGCHART_STYLING") || component === undefined) {
+      return styleValue;
```

Comment:
Yes, it is for backwards compatibility. Taking the values from the old structure.

Reply:
hmm a better name would make it easier to read I think. Not a huge fan of prefixing things with old and new, but maybe call it `depricatedValue`? Since we should also change the api spec for this